Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -103,7 +103,7 @@ extension Message {
   /// - Parameter options: The JSONDecodingOptions to use.
   /// - Throws: `JSONDecodingError` if decoding fails.
   public init(
-    jsonUTF8Data: Data,
```

Comment:
As mentioned above, should still still be "Data" in the name?  As is the PR is inconsistent on when it changed names, so some things are breaking and other aren't.

Reply:
Yes that's true. I'll rename these, but only in the deserialisation path (i.e. I'll keep the `jsonUTF8Data()` methods for now) since serialisation changes will follow in a later PR.
To keep things in line with the comments on binary decoding, I'll copy this method with the old name (i.e. using `jsonUTF8Data:`) into `SwiftProtobufFoundationCompat` to aid in migration.